docker/Dockerfile-netty-centos6

```diff
@@ -0,0 +1,22 @@
+FROM centos:6
```

Comment:
I suggest starting from OpenJDK base image, so that you don't have to do the install yourself.
https://hub.docker.com/r/library/openjdk/tags/

Reply:
we need to ensure we use centos6 so the native stuff is build against the correct GLIBC version etc.

Reply:
I see, that's why I couldn't get it working on Debian. This PR is going to make life easier for a lot of people.